web/ui/react-app/src/pages/graph/DataTable.tsx

```diff
@@ -56,16 +56,18 @@ const DataTable: FC<QueryResult> = ({ data }) => {
     return <Alert color="secondary">Empty query result</Alert>;
   }
 
+  const maxFormattableSize = 100;
   let rows: ReactNode[] = [];
   let limited = false;
+  const toFormatStyle = data.result.length <= maxFormattableSize;
```

Comment:
With the var name `toFormatStyle` I would expect some kind of style object or so, not a boolean. How about calling it `doFormat`?